content/docs/setup/kubernetes/multicluster-install/vpn/index.md

```diff
@@ -28,13 +31,18 @@ across the multicluster environment and may not overlap.
 
 ## Overview
 
-Multicluster functions by enabling Kubernetes control planes running
-a remote configuration to connect to **one** Istio control plane.
+In this mode, multiple Kubernetes control planes running
+a remote configuration connect to a **single** Istio control plane.
 Once one or more remote Kubernetes clusters are connected to the
 Istio control plane, Envoy can then communicate with the **single**
 Istio control plane and form a mesh network across multiple Kubernetes
 clusters.
 
+{{< image width="80%" ratio="36.01%"
```

Comment:
@rshriram Can you provide me with this image? If you do, I can add the diagram to PR #2703 and you can leave the file untouched. I can also move the file to the new folder. Those two things would resolve the conflict between PRs